_episodes/44.md

```diff
@@ -0,0 +1,213 @@
+     time:
+   - title: "Real world use cases"
+     desc: "Trino, Metabase, and more at Meesho"
+     time:
+   - title: "PR of the episode"
+     desc: "Looking at the driver source"
+     time:
+---
+
+## Hosts
+
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+
+## Guests
+
+* [Luis Paolini](https://www.linkedin.com/in/luispaolini/), Product Advocate at
+  [Metabase](https://www.metabase.com/)
+* [Andrew DiBiasio](https://www.linkedin.com/in/andrewdibiasio/), Software
+  Engineer at [Starburst](https://starburst.io)
+* [Piotr Leniartek](https://www.linkedin.com/in/piotrleniartek), Product Manager
+  at [Starburst](https://starburst.io)
+
+## Recap of Trino in 2022
+
+Highlights from the blog post [The rabbit reflects on Trino in 2022]({% post_url
+2023-01-10-trino-2022-the-rabbit-reflects %}):
+
+* 
+
+## Releases 404 to 406
+
+Official highlights from Martin Traverso:
+
+[Trino 404]({{site.url}}/docs/current/release/release-404.html) not found
+
+[Trino 405]({{site.url}}/docs/current/release/release-405.html)
+
+* Support for `ALTER COLUMN ... SET DATA TYPE` statement.
+* Support for Apache Arrow when reading from BigQuery.
+* Support for views in the Delta Lake connector.
+* Support for the Iceberg REST catalog.
+* Support for Protobuf encoding in the Kafka connector.
+* Support for fault-tolerant execution in the MongoDB connector.
+* Support for `DELETE` and query pushdown in the Redshift connector.
+* Performance improvements when reading Parquet data.
+
+[Trino 406]({{site.url}}/docs/current/release/release-406.html)
+
+* tbd
+
+Further thoughts and highlights from Cole:
+
+* tbd
+
+More detailed information is available in the release notes for
+[Trino 405]({{site.url}}/docs/current/release/release-405.html),
+and
+[Trino 406]({{site.url}}/docs/current/release/release-406.html).
+
+We also shipped trino-python-client 0.321.0 with the following improvements:
+
+* Add support for SQLAlchemy 2.0.
+* Add support for `varbinary` query parameters.
+* Add support for variable precision `datetime` types.
+
+## What is Metabase
+
+[Metabase](https://www.metabase.com/) is the BI tool with the friendly UX and
+integrated tooling to let your company explore data on their own.
+
+Core features and advantages of Metabase include the following:
+
+* Visual query build
+* Dashboards
+* Models
+
+Metabase is a web-based application that you run on a server and make it
+available for multiple users. It uses SQL to create queries, reports,
+visualizations, dashboards and more.
+
+Metabase is an open source project licensed under the GNU Affero General Public
+License (AGPL) license. It is written in Clojure and therefore runs on the Java
+virtual machine.
+
+Metabase is also the name of the company, founded in 20xx. It provides an
```

Comment:
@paoliniluis do you have a year or founder info or so?

Reply:
Founded in 2014. The company came out from Expa, one of the most recognized company builders in SF. Metabase is running in more than 50K intances around the world which 2K of those run on our Cloud.

Reply:
(to note: the 50K are the ones we know about, because you can run Metabase without reporting anonymous statistics or even air gapped)

Reply:
Awesome.